.github/workflows/pull-request.yml

```
@@ -312,3 +312,43 @@ jobs:
+      - name: Create .env file for talawa api testing environment
+        run: cp envFiles/.env.devcontainer .env
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+
+      - name: Build Docker image
+        run: |
+          docker buildx build --file ./docker/api.Containerfile --tag talawa_api --target non_production .
+
+      - name: Start Services with Docker Compose
+        run: docker compose up -d
+      
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)